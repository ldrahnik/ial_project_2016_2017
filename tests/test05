A 2 B
B 1 C
C 4 A
C 3 E
E 2 F
E 3 D
F 2 D
D 4 B
B 2 E
