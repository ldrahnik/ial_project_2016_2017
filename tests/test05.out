FloydWarshall: A -(1)> C -(2)> E 
BellmanFord: A -(1)> C -(1)> F -(2)> E
BellmanFord: A -(1)> C -(2)> E
BellmanFord: A -(-1)> D -(1)> F -(2)> E
